pkg/utils/resources/resources.go

```diff
@@ -93,20 +93,94 @@ func Subtract(lhs, rhs v1.ResourceList) v1.ResourceList {
 	return result
 }
 
-// Ceiling calculates the max between the sum of container resources and max of initContainers
-func Ceiling(pod *v1.Pod) v1.ResourceRequirements {
-	var resources v1.ResourceRequirements
+func PodRequests(pod *v1.Pod) v1.ResourceList {
+	reqs := v1.ResourceList{}
+
 	for _, container := range pod.Spec.Containers {
-		resources.Requests = MergeInto(resources.Requests, MergeResourceLimitsIntoRequests(container))
-		resources.Limits = MergeInto(resources.Limits, container.Resources.Limits)
+		containerReqs := container.Resources.Requests
+
+		MergeInto(reqs, containerReqs)
+
 	}
+
+	restartableInitContainerReqs := v1.ResourceList{}
+	initContainerReqs := v1.ResourceList{}
```

Comment:
```suggestion
	maxInitContainerReqs := v1.ResourceList{}
```

I'd personally rename this since I found it confusing as I was reading through this code to understand what variable was a cumulative value (e.g. `restartableInitContainerReqs`) and what was a maximum value (e.g. `initContainerReqs`)